docs/howto/mbed-tools-howto.md

```diff
@@ -202,6 +202,25 @@ Use CMake to build your application:
 
 1. Drag and drop the generated file to your board.
 
+
+## Mbed OS Configuration and building the project
+
+We can use a single command which will configure (set up your target and toolchain) and build the project.
+
+    1. To build and configure:
+
+    ```
+    mbed-tools build -m <target> -t <toolchain>
```

Comment:
Could we give a copy-pastable example, too? Giving a concrete example, instead of a generic one, is quite helpful in many instances.

Reply:
Done